src/components/widgets/NumberWidget.js

```diff
@@ -106,7 +106,9 @@ export default class NumberWidget extends React.Component {
   }
 
   onChange = e => {
-    this.setState({value: e.target.value, displayValue: e.target.value});
+    const value = e.target.value;
+    this.setState({value: value, displayValue: value});
+    this.setValue(value);
```

Comment:
OK. Looking to find a way to let it change `onChange` so you don't have to keep blurring to see the changes.